skopt/gbt.py

```diff
@@ -50,4 +50,4 @@ def fit(self, X, y):
 
     def predict(self, X):
         """Predictions for each quantile."""
-        return np.vstack([rgr.predict(X) for rgr in self.regressors_])
+        return np.asarray([rgr.predict(X) for rgr in self.regressors_])
```

Comment:
Also it is not data-dependent and not argument dependent. It am thinking it might be better to return the approximate std directly..